app/controllers/api/v2/smart_proxies_controller.rb

```diff
@@ -10,16 +10,21 @@ class SmartProxiesController < V2::BaseController
       api :GET, "/smart_proxies/", N_("List all smart proxies")
       param_group :taxonomy_scope, ::Api::V2::BaseController
       param_group :search_and_pagination, ::Api::V2::BaseController
+      param :include, ['status'], N_("Array of extra information types to include")
       add_scoped_search_description_for(SmartProxy)
 
       def index
         @smart_proxies = resource_scope_for_index.includes(:features)
+        if params[:include].present?
+          @status = params[:include].include?('status')
+        end
```

Comment:
I'm not a fan of relying on undefined variables. Isn't this the same?
```suggestion
        @status = params[:include]&.include?('status')
```